src/clearable.js

```diff
@@ -0,0 +1,27 @@
+angular.module( "clearable", [] )
+
+.directive( "clearable", [ "$compile", function ( $compile )
+    {
+        "use strict";
+
```

Comment:
Maybe throw a "usage" comment in here, ala this: https://github.com/Tathanen/vokal-ng-lib/blob/master/directives/vokal_date_picker.js